### ISSUE TITLE
Pinecone Endpoint

### ISSUE THREAD

**Issue by Belyua** [wombyz/HormoziGPT#3]
Hello, I think the repository should be clarified.  What should pinecone_endpoint look like?  does it need to be created on hugging face or is it the same link from the index page on the pinecone site?  does anyone have an example link?

**Comment by augustov58** [comment]
Agree. Unless you know what data you have and know how to embed it and then upload to the Pinecone database this code won't work. he didn't show how to make that in the video
